example.json

```diff
@@ -11,14 +11,34 @@
             "all": "All",
             "document": "PDF",
             "media": "Media",
-            "link": "Link"
+            "link": "Link",
+            "custom1": "Custom category 1",
+            "custom2": "Custom category 2",
+            "custom3": "Custom category 3",
+            "custom4": "Custom category 4",
+            "custom5": "Custom category 5",
+            "custom6": "Custom category 6",
+            "custom7": "Custom category 7",
+            "custom8": "Custom category 8",
+            "custom9": "Custom category 9",
+            "custom10": "Custom category 10"
```

Comment:
Might be worth adding these to the _resourcesItems node too, it threw me off at first!

Reply:
@StuartNicholls I've added an example of using the "custom1" category. That probably gets the idea across?

Reply:
Yeah, lovely